sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2985,7 +2985,7 @@ pi_result cuda_piEnqueueKernelLaunch(
             return PI_ERROR_INVALID_WORK_GROUP_SIZE;
 
           if (local_work_size[dim] > maxThreadsPerBlock[dim])
-            return PI_ERROR_INVALID_WORK_ITEM_SIZE;
+            return PI_ERROR_INVALID_WORK_GROUP_SIZE;
```

Comment:
This is part of https://github.com/intel/llvm/pull/8483, could you please update this PR.

Reply:
Sorry, I tried to remove the content of *8483 from the current PR.